raiden/blockchain_events_handler.py

```diff
@@ -114,6 +112,10 @@ def handle_channel_new(raiden, event, current_block_number):
         )
         raiden.handle_state_change(new_route, current_block_number)
 
+    # A new channel is available, run the connection manager in case more
+    # connections are needed
+    gevent.spawn(connection_manager.retry_connect)
```

Comment:
`connection_manager` isn't defined at this point, you need to move the definition in [line 92](https://github.com/raiden-network/raiden/blob/290568f38b0e8683183dfc4890f04aeb6a352fae/raiden/blockchain_events_handler.py#L92) out of the `if`
https://github.com/raiden-network/raiden/blob/290568f38b0e8683183dfc4890f04aeb6a352fae/raiden/blockchain_events_handler.py#L92